libs/wingsdk/src/target-tf-aws/app.ts

```diff
@@ -68,8 +83,126 @@ export class App extends CdktfApp {
 
       case TOPIC_FQN:
         return new Topic(scope, id, args[0]);
+
+      case REDIS_FQN:
+        return new Redis(scope, id);
     }
 
     return undefined;
   }
+
+  /**
+   * Returns the VPC for this app. Will create a new VPC if one does not exist.
+   */
+  public get vpc(): Vpc {
+    if (this._vpc) {
+      return this._vpc;
+    }
+
+    const VPC_NAME_OPTS: NameOptions = {
+      maxLen: 32,
+      disallowedRegex: /[^a-zA-Z0-9-]/,
+    };
+
+    const identifier = ResourceNames.generateName(this, VPC_NAME_OPTS);
+
+    // create the app wide VPC
+    this._vpc = new Vpc(this, "VPC", {
+      cidrBlock: "10.0.0.0/16",
+      enableDnsHostnames: true,
+      enableDnsSupport: true,
+      tags: {
+        Name: `${identifier}-vpc`,
+      },
+    });
+
+    // Create the subnets for the VPC, in order to ensure internet egress there
+    // is a minimum requirement of 2 subnets, one public and one private. As well
+    // as a NAT gateway and internet gateway. The NAT gateway is required to
+    // allow the private subnet to route traffic to the internet. The internet
+    // gateway is required to allow the NAT gateway to route traffic to the
+    // internet.
```

Comment:
Specifically for redis, why do we need internet access/EIP/public subnet etc..?

Reply:
For Redis we don’t  but for functions that will use Redis and be deployed into the private subnet we require this.